public/package.json

```diff
@@ -4,31 +4,20 @@
   "private": true,
   "devDependencies": {
     "babel-core": "^6.24.1",
-    "babel-plugin-react-transform": "^3.0.0",
     "babel-plugin-transform-react-jsx": "^6.24.1",
-    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-env": "1.7.0",
     "babel-preset-react": "^6.24.1",
-    "deep-diff": "^1.0.0-pre.2",
     "grunt": "^1.0.1",
     "grunt-babel": "^7.0.0",
     "grunt-cli": "^1.2.0",
     "grunt-contrib-cssmin": "^2.2.1",
     "grunt-contrib-less": "^1.4.1",
     "grunt-contrib-watch": "^1.0.0",
-    "grunt-jsx": "^0.1.8",
-    "grunt-react": "^0.12.3",
-    "grunt-reactjsx": "^0.2.4",
     "jasmine": "^2.8.0",
-    "js-yaml": "^3.11.0",
     "jsdom": "^13.0.0",
     "karma": "^2.0.0",
     "karma-jasmine": "^1.1.1",
     "karma-jsdom-launcher": "^6.1.4",
-    "karma-junit-reporter": "^1.2.0",
-    "load-grunt-tasks": "^3.5.2",
-    "numeral": "^2.0.6",
-    "react": "^16.2.0",
-    "react-bootstrap": "^0.31.5",
-    "react-dom": "^16.2.0"
```

Comment:
just wanted to double check an assumption I was making: so is it like these dependencies didn't actually do anything because we have them copied in the thirdparty folder, and we're actually using those minified copies?